src/appkit.cs

```diff
@@ -28450,5 +28426,87 @@ interface NSImageSymbolConfiguration : NSCopying, NSSecureCoding
 		[Mac (12,0)]
 		[Export ("configurationByApplyingConfiguration:")]
 		NSImageSymbolConfiguration Create (NSImageSymbolConfiguration configuration);
+
+		[Mac (13,0)]
+		[Static]
+		[Export ("configurationPreferringMonochrome")]
+		NSImageSymbolConfiguration GetConfigurationPreferringMonochrome ();
```

Comment:
All the other methods in this class seem to use Create:

```suggestion
		NSImageSymbolConfiguration CreateConfigurationPreferringMonochrome ();
```